### PR TITLE
Update api doc of vote status endpoint

### PR DESCRIPTION
**Pull request by Mr-Leshiy**

### CODE REVIEW

File: doc/api/v0.yaml [hunk 1]
```diff
@@ -2215,6 +2215,16 @@ paths:
                   description: Vote Plan
                   type: object
                   properties:
+                    id:
+                      type: string
+                      description: Hex-encoded vote plan ID
+                      pattern: '[0-9a-f]+'
+                      minLength: 64
+                      maxLength: 64
```
Comment (eugene-babichenko) [inline doc/api/v0.yaml:2223]: Could be just `[0-9a-f]{64}`

**Review by eugene-babichenko — CHANGES_REQUESTED** [review]

File: doc/api/v0.yaml [hunk 2]
```diff
@@ -2215,6 +2215,14 @@ paths:
                   description: Vote Plan
                   type: object
                   properties:
+                    id:
+                      type: string
+                      description: Hex-encoded vote plan ID
+                      pattern: '[0-9a-f]+{64}'
```
Comment (Zeegomo) [inline doc/api/v0.yaml:2221]:
```suggestion
                      pattern: '[0-9a-f]{64}'
```

File: doc/api/v0.yaml [hunk 3]
```diff
@@ -2227,23 +2235,28 @@ paths:
...
+                            minimum: 0
+                            maximum: 255
+                          proposal_id:
                             type: string
-                            description: Hash of external id
-                            pattern: '[0-9a-f]+'
-                            minLength: 64
-                            maxLength: 64
+                            description: Hex-encoded proposal id
+                            pattern: '[0-9a-f]+{64}'
```
Comment (Zeegomo) [inline doc/api/v0.yaml:2259]:
```suggestion
                            pattern: '[0-9a-f]{64}'
```

File: doc/api/v0.yaml [hunk 4]
```diff
@@ -2259,36 +2272,130 @@ paths:
...
+                                                        minimum: 0
+                                                        maximum: 15
+                                                      end:
+                                                        type: integer
+                                                        description: upper bound of the available options, maximum being 16 (not included)
+                                                        minimum: 1
+                                                        maximum: 15
+                          votes_cast:
+                            type: integer
+                            description: votes vast
```
Comment (Zeegomo) [inline doc/api/v0.yaml:2355]:
```suggestion
                            description: votes cast
```

File: doc/api/v0.yaml [hunk 5]
```diff
@@ -2259,36 +2272,130 @@ paths:
...
+                                            type: object
+                                            properties:
+                                              start:
+                                                type: integer
+                                                description: starting value of the vote option, starting from 0
+                                                minimum: 0
+                                                maximum: 15
+                                              end:
+                                                type: integer
+                                                description: upper bound of the available options, maximum being 16 (not included)
```
Comment (Zeegomo) [inline doc/api/v0.yaml:2306]: I think the maximum is actually 255

File: doc/api/v0.yaml [hunk 6]
```diff
@@ -2259,36 +2272,130 @@ paths:
...
+                                                      start:
+                                                        type: integer
+                                                        description: starting value of the vote option, starting from 0
+                                                        minimum: 0
+                                                        maximum: 15
+                                                      end:
+                                                        type: integer
+                                                        description: upper bound of the available options, maximum being 16 (not included)
+                                                        minimum: 1
+                                                        maximum: 15
```
Comment (Zeegomo) [inline doc/api/v0.yaml:2352]: this could be abstracted to avoid repetition between public and private tally

File: doc/api/v0.yaml (hunk 5 above)
Comment (Mr-Leshiy) [reply doc/api/v0.yaml:2306]: Are you sure ? An upper bound for the `start` is correct ?

Comment (Zeegomo) [reply doc/api/v0.yaml]: You're right, but I swear I remember there was a limit of 256 regarding choices somewhere.
Maybe it's just a vote that can be constructed for up to 256 options, but it's not currently allowed to create a proposal with that many choices

File: doc/api/v0.yaml (hunk 6 above)
Comment (Mr-Leshiy) [reply doc/api/v0.yaml]: Updated

**Pull request merged**